infra/quickstart/formbricks.sh

```
@@ -0,0 +1,313 @@
+
+  # Create alias for kubectl
+  echo "🔧 Creating kubectl alias..."
+  sudo snap alias microk8s.kubectl kubectl >/dev/null 2>&1
+
+  # Wait for microk8s to be ready
+  echo "⏳ Waiting for microK8s to be ready..."
+  sudo microk8s status --wait-ready >/dev/null 2>&1
+
+  # Setting up microK8s configuration
+  mkdir -p ~/.kube
+  sudo microk8s config > ~/.kube/config
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)